lib/logstash/bundler.rb

```diff
@@ -0,0 +1,125 @@
+require "bundler"
```

Comment:
Can we call this `bundler_patches.rb` ?

Maybe we should have a `lib/logstash/patches/...` directory in which we put `lib/logstash/patches/bundler.rb` ?